stage0/run.go

```diff
@@ -211,6 +204,29 @@ func Run(dir string, debug bool) {
 	}
 }
 
+func makeUniqueContainer(pdir string) (*types.UUID, string, error) {
```

Comment:
Can you also add a comment telling us that the second return value is a full path to the new directory?